File: javascript/nivel 3 listas textos/19.js
/*
Ejercicio 19
Ordenar una lista sin usar sort()
(Método burbuja)
*/
let numeros = [5,2,9,1,5,6];
let n = numeros.length;

for (let i = 0; i < n; i++) {
  for (let j = 0; j < n - i - 1; j++) {
    if(numeros[j] > numeros[j + 1]){
      //intercambio
      let temp =numeros[j];
      numeros[j] = numeros[j +1];
      numeros[j + 1]= temp;
    }
  }
}

console.log("Lista ordenada:", numeros);

File: python/nivel 3 listas textos/19.py
"""
Ejercicio 19
Ordenar una lista sin usar sort()
(Método burbuja)
"""
numeros = [5,2,9,1,5,6]

n = len(numeros)

for i in range(n):
  for j in range(0, n - i - 1):
    if numeros[j] > numeros[j + 1]:
      numeros[j], numeros[j + 1] = numeros[j + 1],numeros[j]

print("la lista ordenada es: ",numeros)
